locale/en/knowledge/REPL/how-to-create-a-custom-repl.md

```diff
@@ -8,93 +8,99 @@ difficulty: 2
 layout: knowledge-post.hbs
 ---
 
-Node allows users to create their own REPLs with the [repl module](https://nodejs.org/docs/v0.4.10/api/repl.html). Its basic use looks like this:
+Node allows users to create their own REPLs with the [repl module](https://nodejs.org/api/repl.html). Its basic use looks like this:
```

Comment:
Since we are here, maybe it's a good time to change `Node` to `Node.js`

```suggestion
Node.js allows users to create their own REPLs with the [repl module](https://nodejs.org/api/repl.html). Its basic use looks like this:
```

For your information: 

![Screen Shot 2019-03-20 at 11 16 24 AM](https://user-images.githubusercontent.com/23313266/54656704-ad890780-4b01-11e9-8698-78f89522edaa.png)

Reply:
Fixed with help :)